src/main/java/seedu/taassist/model/session/Session.java

```diff
@@ -0,0 +1,69 @@
+package seedu.taassist.model.session;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.taassist.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Session for a {@code ModuleClass} in TA-Assist.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidSessionName(String)}
+ */
+public class Session {
+    public static final String MESSAGE_CONSTRAINTS = "Session names can take any values, but it should not be blank";
```

Comment:
tbh I know message for Address class is same as this, but it seems like there's some grammar mistakes, do we correct them?

Reply:
I don't see a grammar mistake. :thinking:

Reply:
@optionalemon is the grammar mistake being `values` instead of `value`? That is what I'm seeing.